com.ibm.streamsx.topology/opt/python/packages/streamsx/spl/types.py

```diff
@@ -183,3 +183,11 @@ def rstring(value):
     Create an SPL ``rstring`` value.
     """
     return Expression('RSTRING', str(value))
+
+_null = Expression.expression("null")
+
+def valueNull():
```

Comment:
Python does not use camelcase for function names, but why not simply have the field be `NULL` and no need for a function? Or have the function be `null()`.